CONSOLE MESSAGE: XMLHttpRequest cannot load http://nonexistent-origin.localhost}:8800. Cross origin requests are only supported for HTTP.

FAIL XmlHttpRequest: The send() method: Throw a "throw an "NetworkError" exception when Network error happens (synchronous flag is set) assert_throws: function "function () {
                xhr.send("Test Message");
 ..." threw object "Error: NETWORK_ERR: XMLHttpRequest Exception 101" that is not a DOMException NetworkError: property "code" is equal to 101, expected 19(stack: assert@http://localhost:8800/resources/testharness.js:1738:42
assert_throws@http://localhost:8800/resources/testharness.js:682:23
http://localhost:8800/XMLHttpRequest/send-network-error-sync-events.sub.htm:22:26
step@http://localhost:8800/resources/testharness.js:798:30
test@http://localhost:8800/resources/testharness.js:73:22
global code@http://localhost:8800/XMLHttpRequest/send-network-error-sync-events.sub.htm:16:13)

